tests/workflows/spec.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+from pathlib import Path
+
+from halfpipe.ingest.events import ConditionFile
+from halfpipe.model.feature import FeatureSchema
+from halfpipe.model.file.base import File
+from halfpipe.model.file.schema import FileSchema
+from halfpipe.model.setting import SettingSchema
+from halfpipe.model.spec import Spec, SpecSchema
+from halfpipe.resource import get as get_resource
+from templateflow.api import get as get_template
+
+
+def make_spec(
+    dataset_files: list[File],
+    pcc_mask: Path,
+    event_file: File | None = None,
```

Comment:
I just noticed that we didn't consider that the `make_spec` function will not need a separate `events_file` argument if the input data is in BIDS format, because then the event files will be loaded in automatically.
I would suggest instantiating a `Database` and querying for event files instead, that should be the most robust option.

Reply:
Could you provide some context on why you resolved this?

Reply:
Resolved for now as we don't have a task based feature yet